api/lib/block-listeners/base-node-subscription.js

```diff
@@ -124,9 +139,92 @@ class BaseNodeSubscription {
       this.store.update({
         validators: validators
       })
+      const storeValidators = Object.values(this.store.validators)
+      if (storeValidators.length > 0) {
```

Comment:
in which case would there be no validators?

Reply:
on API launch

Reply:
maybe there is an await missing?

Reply:
ok, it was that. Fixed :ok_hand: